node/src/components/small_network.rs

```diff
@@ -186,17 +177,19 @@ where
     /// This is not incoming bandwidth but an independent resource estimate.
     #[data_size(skip)]
     incoming_limiter: Box<dyn Limiter>,
+
+    /// The era that is considered the active era by the small network component.
+    active_era: EraId,
+
+    /// The validator set associated with the `active_era`.
+    active_era_validators: HashSet<PublicKey>,
```

Comment:
So, we might want two of these. 

Specifically `active_era_validators` and `next_era_validators` (which are the validators for `active_era` + 1).

I did suggest having the `last_era_validators` (for `active_era` - 1).  But after some thought I don't think they are necessary.

WDYT?

It doesn't matter *too* much in practice, since validators don't rotate very much, and this is KISS which I approve.

Reply:
The limiters do not make the assumption that they are always going to be called "in order", so the cannot rely on the previous set of validators being the ones right before the new set. There would have to be more logic in the limiters to handle this and/or an active era tracked, but I would rather keep them "dumb".

Reply:
Removed in c5720c5